esy/Package.ml

```diff
@@ -53,6 +53,34 @@ module CommandList = struct
 
 end
 
+(**
+ * Scripts with keys as specified in "scripts".
+ *)
+module Scripts = struct
+
+  type script = {
+    key : string;
+    command : string;
+  }
+  [@@deriving (show, eq, ord)]
+
+  type t =
+    script list
+    [@@deriving (show, eq, ord)]
+
+  let of_yojson = function
```

Comment:
If we change to map repr then we can use `Json.Parse.stringMap` parser combinator to write `of_yojson`.